cpp/net/url_fetcher.cc

```diff
@@ -245,6 +246,8 @@ void State::RequestDone(evhtp_request_t* req) {
               body_length);
   response_->body.swap(body);
 
+  VLOG(2) << *response_;
```

Comment:
Yeah, there's an `operator<<` defined for that so it'll print all the fields nicely formatted.